website/reference.py

```diff
@@ -276,6 +276,32 @@ def update():
         ui.button('Update', on_click=update)
         ui.button('Select all', on_click=lambda: table.call_api_method('selectAll'))
 
+    @example(ui.qtable, menu)
+    def qtable_example():
+        fields = [
+            {'name': 'name', 'field': 'name', 'label': 'Name', 'required': True},
+            {'name': 'age', 'label': 'Age', 'field': 'row => row.age.deep', 'sortable': True},
```

Comment:
Let me suggest that this should be done in python. 
Instead of using a JS method client side, prepare your data accordingly server-side in python.